ui/src/containers/NodeVolumes.js

```diff
@@ -95,11 +229,47 @@ const NodeVolumes = props => {
     volumeSortedList = sortSelector(volumeSortedList, sortBy, sortDirection);
   }
 
+  const onClickDeleteButton = deleteVolumeName => {
+    props.deleteVolume(deleteVolumeName);
+    setisDeleteConfirmationModalOpen(false);
+  };
+
+  const onClickCancelButton = () => {
+    setisDeleteConfirmationModalOpen(false);
+  };
+
   return (
     <>
+      <Modal
```

Comment:
- It would be nice to create a reusable component and It should be moved to ```core-ui```.
something like:
```
<Modal>
    <ModalTitle>{props.title}</ModalTitle>
    <ModalBody>{props.body}</ModalBody>
    <ModalFooter>{props.footer}</ModalFooter>
</Modal>
```
- we need to have only one instance of "Modal" per application like Notifications, but it can be managed later.

Reply:
Will improve it later. Thanks